.travis.yml

```diff
@@ -101,7 +101,7 @@ jobs:
         ENABLED_INTEGRATIONS="cassandra kerberos mongo openldap rabbitmq redis"
         RUN_INTEGRATION_TESTS=all
       stage: test
-      script: ./scripts/ci/ci_run_airflow_testing.sh tests/tests_core.py
+      script: ./scripts/ci/ci_run_airflow_testing.sh tests/test_core.py
```

Comment:
```suggestion
```
That's an interesting one. Actually it's a mistake here. We should not constraint ourselves in this test only to test_core. It is likely a remnant from some testing done when I was implementing it. Good that you noticed it.

Reply:
fixed!!! https://github.com/apache/airflow/pull/7396/commits/f6be34fd8725ae44b156f0e51076d92bf69a36a0